getting-started/setup-building.rst

```diff
@@ -58,6 +58,24 @@ itself. Git is easily available for all common operating systems.
   ``git push``, or ``git fetch``.  On Windows, you should also
   :ref:`enable autocrlf <autocrlf>`.
 
+.. _install-pre-commit:
+
+Install pre-commit
+------------------
+
+To make sure your code is formatted correctly, we recommend installing
```

Comment:
pre-commit runs linting as well as formatting, and formatting is more limited in scope. Shall we just say "linted" here?

```suggestion
To make sure your code is linted correctly, we recommend installing
```